api/src/aggregator/aggregator.controller.ts

```diff
@@ -81,6 +89,26 @@ export class AggregatorController {
       throw new HttpException(msg, HttpStatus.NOT_FOUND)
     }
 
+    // verify aggregator on load
+    const newAggregatorDto: AggregatorDto = {
+      aggregatorHash: aggregatorRecord.aggregatorHash,
+      active: aggregatorRecord.active,
```

Comment:
I don't think you should be passing `active` property there.

Reply:
To able to create new `AggregatorDto` we need to pass `active` property. [Link](https://github.com/Bisonai/orakl/blob/i-339/feat/validate-adapter-and-aggregator-throught-api/api/src/aggregator/dto/aggregator.dto.ts#L7)

I think it won't make any problem since we remove `active` property when we calculate hash. [Link](https://github.com/Bisonai/orakl/blob/i-339/feat/validate-adapter-and-aggregator-throught-api/api/src/aggregator/aggregator.service.ts#L131-L135)

```typescript
    // Don't use following properties in computation of hash
    delete input.aggregatorHash
    delete input.address
    delete input.active
```

Reply:
Okay! Thank you!